pkg/volume/azure_file/azure_file.go

```diff
@@ -207,7 +209,12 @@ func (b *azureFileMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 	os.MkdirAll(dir, 0750)
-	source := fmt.Sprintf("//%s.file.core.windows.net/%s", accountName, b.shareName)
+
+	azure, err := getAzureCloud(b.plugin.host.GetCloudProvider())
```

Comment:
@karataliu @rootfs The cloud environment identifier would have to be provided to the azure_file plugin independently of the azure cloud provider. What are your thoughts on this?